### ISSUE TITLE
ctc_loss misses logits and targets spatial dim args

### ISSUE THREAD

**Issue by albertz** [rwth-i6/returnn_common#239]
Internally it uses time_dim_axis, and it also only expects {B,T,D} as shape.
